layouts/partials/google-analytics.html

```diff
@@ -1,13 +0,0 @@
-<!-- Google Analytics -->
```

Comment:
I thought the same thing. But, it appears the reference in the `head_includes.html` is referencing an internal file inside the Hugo project. Because, when I would make changes to `google-analytics.html` they would not appear in my `exampleSite`. And then--once I got the analytics snippets to appear--I deleted `google-analytics.html` and the analytics snippets still was still visible. All that lead me to believe `layouts/partials/google-analytics.html` wasn't needed.